### PR TITLE
Moves Leaflet-IIIF away from percent sizes in tile requests. Using pe…

### PR DESCRIPTION
**Pull request by mejackreed**
…rcents in size requests is compliant with the IIIF specification, but some IIIF servers don't handle this correctly.

Adds an example of an improperly handled pct param now being shown correctly.

cc @azaroth42

### CODE REVIEW

**Comment by mejackreed** [comment]
Hey @eliotjordan ... any implications for you?

**Comment by eliotjordan** [comment]
@mejackreed Very nice. Everything works perfectly with our IIIF servers.

File: leaflet-iiif.js [hunk 1]
```diff
@@ -30,15 +30,32 @@ L.TileLayer.Iiif = L.TileLayer.extend({
...
+  * a tile
+  * @param {Number} x - The width of a tile
+  * @param {Number} y - The height of a tile
+  * @returns {String}
+  */
+  _iiifSizeParam: function(x, y) {
+    if (x >= y) {
+      return x + ',';
+    } else {
+      return ',' + y;
```
Comment (azaroth42) [inline leaflet-iiif.js]: The ,h format is not the canonical one... it should always be width, for size:
  http://iiif.io/api/image/2.0/#canonical-uri-syntax 

If you know the width of the tile, the most performant approach is to always use that and let the server calculate the height (as that's the URI most likely to be cached)

But a minor issue -- it's a big improvement over using pct :)

Comment (mejackreed) [inline leaflet-iiif.js]: Thanks... modified the comment to remove "canonical". Though the canonical way seems best here, in testing several servers had difficulty when only supplying the `w,` or supplying `w,h`.

Looks like this logic will need to stay in there for now.

**Comment by eliotjordan** [comment]
Let's do this. :boat: :boat:

**Pull request merged**